src/lib/IbexaProductVersion.php

```diff
@@ -41,7 +41,17 @@ public static function getInstalledProduct(): string
         );
 
         // first element in the array is the package matching product edition
-        return reset($installedIbexaProducts);
+        $installedProduct = reset($installedIbexaProducts);
+
+        if (!is_string($installedProduct)) {
+            throw new \RuntimeException(sprintf(
```

Comment:
Nitpick: Missing `\RuntimeException` import

Reply:
Fixed in 40069d0.